TCP_Server.py

```diff
@@ -82,8 +83,8 @@ def on_read(self, conn, mask):
             while True:
                 limit -= 1
                 data += conn.recv(1).decode()
-                if '\x04' in data or limit == 0:
-                    data = data.split('\x04')[0]
+                if self.EOT in data or limit == 0:
+                    data = data.split(self.EOT)[0]
```

Comment:
If you move EOT to connection interface then it can return message without EOT char.